components/Header/Header.tsx

```diff
@@ -80,6 +80,10 @@ function Header() {
           ? 'ROPSTEN'
```

Comment:
Ropsten is unavailable now. We need to remove it

Reply:
Can fix in the next iteration